tests/test_other.py

```diff
@@ -368,11 +367,8 @@ def test_emcc_4(self, compiler):
         # closure has not been run, we can do some additional checks. TODO: figure out how to do these even with closure
         assert '._main = ' not in generated, 'closure compiler should not have been run'
         if keep_debug:
-          assert ('switch (label)' in generated or 'switch (label | 0)' in generated) == (opt_level <= 0), 'relooping should be in opt >= 1'
-          assert ('assert(STACKTOP < STACK_MAX' in generated) == (opt_level == 0), 'assertions should be in opt == 0'
+          assert ('assert(INITIAL_INITIAL_MEMORY >= TOTAL_STACK' in generated) == (opt_level == 0), 'assertions should be in opt == 0'
```

Comment:
hmm, what is `INITIAL_INITIAL_MEMORY` - ? I see it's in `src/preamble.js` but seems like it's a typo maybe and should be just `INITIAL_MEMORY`?

Reply:
Oh.. i didn't even think about that .. I was just searching for any assert that looked stable.  I will revisit..

Reply:
Actually INITIAL_INITIAL_MEMORY is really a thing... its generated by src/preamble.js

Reply:
Yes, but I think it's a typo there? We should probably fix it there (and now here).

Reply:
If it is a type then its repeated in quite a few different places and should and easy search/replace to fix.   I don't think landing this change makes it any harder to fix that as a separate issue

Reply:
I agree, it's a separate issue. Anyhow, I'll fix it.